data/data/aws/variables-aws.tf

```diff
@@ -53,12 +53,12 @@ EOF
 }
 
 variable "aws_master_root_volume_type" {
-  type        = string
+  type = string
   description = "The type of volume for the root block device of master nodes."
 }
 
 variable "aws_master_root_volume_size" {
-  type        = string
+  type = string
```

Comment:
seems like something weird is going on with tf-fmt, or I'm missing something... normally tf-fmt likes to have the = vertically aligned. I encountered something similar.

Reply:
yeah, i think we are using an old version of tf that had some weird bug... we should fix it and use a newer version..